lib/features/thread/presentation/widgets/email_tile_web_builder.dart

```diff
@@ -46,6 +50,24 @@ class EmailTileBuilder {
     _onMoreActionClick = onMoreActionClick;
   }
 
+  void _onHoverIconChanged(bool isHover, StateSetter setState) {
+    if (_debounceTimeIcon?.isActive ?? false) _debounceTimeIcon?.cancel();
+    _debounceTimeIcon = Timer(const Duration(milliseconds: 300), () {
+      setState(() {
```

Comment:
we don't have other way to do that? with `setState` here, IMO we maybe need to re-render all MailboxDashboard 😨

Reply:
Do you understand what setState is here?. Please read more code signing.